src/nvim/ex_cmds.h

```diff
@@ -26,6 +29,26 @@ typedef struct {
   list_T *additional_elements;  ///< Additional data left from ShaDa file.
 } SubReplacementString;
 
+
+// Defs for inc_sub functionality
+
+// List of matches on a single line
+typedef kvec_t(colnr_T) ColNrVec;
```

Comment:
this one you could still remove, as it's part of a bigger struct and not directly addressed, unlike `MatchedLineVec`